harness/propertyHelper.js

```diff
@@ -1,5 +1,6 @@
 
 function isConfigurable(obj, name) {
+    var desc = Object.getOwnPropertyDescriptor(obj, name);
```

Comment:
Tests can't be run in isolation if they're going to be usefully run in a browser context - this seems like a philosophical concern that affects the entirety of the test262 process.

In other words, it's going to be impractical to run every test in a separate realm in a non-command-line context :-/